samples/sample_common/src/sample_utils.cpp

```diff
@@ -1213,6 +1213,109 @@ mfxStatus CSmplYUVWriter::WriteNextFrameI420(mfxFrameSurface1 *pSurface)
     return MFX_ERR_NONE;
 }
 
+QPFileReader::QPFileReader()
```

Comment:
I'd better use in place defaults instead of explicit trivial constructor
```
private:
    mfxU32                                  m_nframes       = 0;
    mfxU32                                  m_cur_frame_num = 0;
    std::vector<QPFileReaderFrameInfo>      m_frame_info;
```